src/pages/leader-board/leader-board.scss

```diff
@@ -0,0 +1,61 @@
+.qcg-leader-board-page {
+    font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+    text-transform: uppercase;
+    overflow-y: scroll;
+    line-height: normal;
+    height: 80vh;
```

Comment:
change the height to height: calc(100vh - 10em); as we did on the other pages for the scroll to work properly.

Reply:
👍